test/api_compatibility.js

```diff
@@ -7,45 +7,17 @@ var expect = require('chai').expect;
 // library.  Again, globals are used purely for convenience.
 var opentracing = require('../debug.js');
 
-module.exports = function apiCompatibilityChecks(createTracer) {
-    describe('OpenTracing API', function() {
-        describe('Constants', function() {
-            var constStrings = [
-                'FORMAT_TEXT_MAP',
-                'FORMAT_BINARY',
-                'FORMAT_HTTP_HEADERS',
-                'REFERENCE_CHILD_OF',
-                'REFERENCE_FOLLOWS_FROM',
-            ];
-            _.each(constStrings, function(name) {
-                it(name + ' should be a constant string', function() {
-                    expect(opentracing[name]).to.be.a('string');
-                });
-            });
-        });
-
-        describe('Standalone functions', function() {
-            var funcs = [
-                'childOf',
-                'followsFrom',
-                'initGlobalTracer',
-                'globalTracer',
-
-            ];
-            _.each(funcs, function(name) {
-                it(name + ' should be a function', function() {
-                    expect(opentracing[name]).to.be.a('function');
-                });
-            });
-        });
+module.exports = function apiCompatibilityChecks(_createTracer) {
+    var createTracer = _createTracer || function() { return opentracing.Tracer(); };
 
+    describe('OpenTracing API', function() {
         describe('Tracer', function() {
             it('should be a class', function() {
-                expect(opentracing.Tracer).to.be.a('function');
-                expect(new opentracing.Tracer()).to.be.an('object');
+                expect(createTracer).to.be.a('function');
+                expect(createTracer()).to.be.an('object');
             });
 
-            var tracer = new opentracing.Tracer();
+            var tracer = new createTracer();
```

Comment:
is `new` appropriate here? I would've expected the function itself to return the tracer already.